tox.ini

```diff
@@ -16,6 +16,7 @@ deps =
   ansible29: ansible>=2.9,<2.10
   ansibledevel: git+https://github.com/ansible/ansible.git
   ruamel.yaml==0.16.5  # NOTE: 0.15.34 has issues with py37
+  pathlib2; python_version < '3.2'
```

Comment:
This shouldn't be necessary as this dep should come from the dist.

Reply:
I agree with you but it does not! Please try it! My guess is that setup.cfg is not used at all due to removal of of setup.py — in fact we should not even see the ruamel here, all deps should be in a single place.

Reply:
Do you have evidence?

Reply:
I will do a test, I will comment the line and see what CI tells us.